python/flexflow/core/flexflow_pybind11.py

```diff
@@ -21,11 +21,64 @@
 from .flexflow_pybind11_internal import begin_flexflow_task, finish_flexflow_task
 from .flexflow_pybind11_internal import Initializer, GlorotUniformInitializer, UniformInitializer, ZeroInitializer
 from .flexflow_pybind11_internal import Optimizer, SGDOptimizer, AdamOptimizer
-from .flexflow_pybind11_internal import Op, NetConfig, SingleDataLoader, Tensor, FFConfig, PerfMetrics
+from .flexflow_pybind11_internal import NetConfig, SingleDataLoader, Tensor, FFConfig, PerfMetrics
+
+from .flexflow_pybind11_internal import Op as _Op
+from .flexflow_pybind11_internal import Parameter as _Parameter
 from .flexflow_pybind11_internal import FFModel as _FFModel
 
 ff_tracing_id = 200
 
+# -----------------------------------------------------------------------
+# Op
+# -----------------------------------------------------------------------
+class Op(object):
+  def __init__(self, op):
+    self.op = op
```

Comment:
@msbaines 
There are 2 issues in this PR. 
1. Classes like Op and Parameter can not inherit from the Op and Parameter (named _Op and _Parameter here) in bindings.cc, because Op is returned from FFModel.get_layer_by_id, which is an instance of _Op, and I can not cast Op into _Op if Op is inherited from _Op.
2. In CFFI, I track the operator type of each operator, so that I can protect the code by allowing certain type of operators to have get_weight_tensor and get_bias_tensor functions, but in pybind, we do not have such information.

Reply:
Instead of sub-classing why don't we just add python methods to pybind class directly:

https://mgarod.medium.com/dynamically-add-a-method-to-a-class-in-python-c49204b85bd6

If we add binding for the individual operators, each operator will have a proper python type.